### ISSUE TITLE
Ability to change Priority Vote?

### ISSUE THREAD

**Issue by tjhunt1** [icon-community/CPS#28]
### Discussed in https://github.com/icon-community/CPS/discussions/27

<div type='discussions-op-text'>

<sup>Originally posted by **tjhunt1** July 21, 2022</sup>
Should we allow Validators to change their Priority Vote once per cycle, similar to how they can for their voting decisions on proposals/progress reports?</div>

**Comment by sdpisreddevil** [comment]
Yeah, give option to change the priority when you can change the vote on Proposals.
